tests/utils/test_filters.py

```diff
@@ -6,278 +6,440 @@
 
 
 class TestDimension:
-
     def test_dimension(self):
-        d = filters.Dimension('dim')
-        actual = filters.Filter.build_filter(d == 'val')
-        expected = {'type': 'selector', 'dimension': 'dim', 'value': 'val'}
+        d = filters.Dimension("dim")
```

Comment:
Sorry would you mind disabling `black` formatting (or similar) when saving this file as it's hard to easily determine what your change encompasses. I sense `black` will probably be enabled for the entire repo in the foreseeable future.

Reply:
I understand. I'll revert black in this PR.
And how about we create issue about apply `black`?

Reply:
@SongYunSeop there should definitely be a PR which formats the entire code using `black` and adds a CI test to enforce all code is `black` formatted going forward.

Reply:
@john-bodley I created issue(#211) and PR(#212) 😎